streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceivedBlockHandler.scala

```diff
@@ -32,7 +32,10 @@ import org.apache.spark.{Logging, SparkConf, SparkException}
 
 /** Trait that represents the metadata related to storage of blocks */
 private[streaming] trait ReceivedBlockStoreResult {
-  def blockId: StreamBlockId  // Any implementation of this trait will store a block id
+  // Any implementation of this trait will store a block id
+  def blockId: StreamBlockId
+  // Any implementation of this trait will have to return the number of records
+  def numRecords: Option[Long]
```

Comment:
Ah, ok. I just find the `num*` method calls weird, when it could be called `*count`. But if it is consistent with everything else, then it is fine.